router.go

```diff
@@ -15,17 +15,19 @@ import (
 	"github.com/checkmarble/marble-backend/utils"
 )
 
-func corsOption(env string) cors.Config {
-	allowedOrigins := []string{"https://app*.checkmarble.com", "https://backoffice*.checkmarble.com"}
+func corsOption(conf AppConfiguration) cors.Config {
+	allowedOrigins := []string{
+		"https://" + conf.config.MarbleAppHost,
+		"https://" + conf.config.MarbleBackofficeHost,
+	}
 
-	if env == "development" {
+	if conf.env == "development" {
 		allowedOrigins = append(allowedOrigins,
 			"http://localhost:3000", "http://localhost:3001", "http://localhost:3002", "http://localhost:5173")
 	}
 
 	return cors.Config{
-		AllowOrigins:  allowedOrigins,
-		AllowWildcard: true,
```

Comment:
no longer needed as each env will have its own specific values (which is a better practice anyway)